bubuku/zookeeper/__init__.py

```diff
@@ -372,3 +467,120 @@ def unregister_change(self, name):
 def load_exhibitor_proxy(address_provider: AddressListProvider, prefix: str) -> BukuExhibitor:
     proxy = _ZookeeperProxy(address_provider, prefix)
     return BukuExhibitor(proxy)
+
+
+class RebalanceThrottleManager(object):
+    _instance = None
+
+    def __new__(cls, zk, throttle, ongoing):
+        if not cls._instance:
+            return cls._RebalanceThrottleManager(zk, throttle, ongoing)
+        cls._instance.throttle = int(throttle)
+        cls._instance.zk = zk
+        cls._instance.ongoing = ongoing
+        return cls._instance
```

Comment:
I do this to maintain a singleton object which stores the current throttle value which gets applied multiple times (at each partition reassignment task), and this throttle value can be updated with the `--ongoing` flag. This would mean that the throttle changes for the current and the future partition reassignment tasks. Now, to remove the configuration from only certain brokers, I maintain the throttled brokers and topics in the same object.

Reply:
Right, from the discussion, understood the point and removed `ongoing` flag in `de8b7c8`. Will open another pull request for it.